apps/video/style/video.css

```diff
@@ -443,21 +442,21 @@ header .icon:after {
 /* video player controls */
 #videoControls {
   position: absolute;
+  display: block;
   top: 0;
   left: 0;
   width: 100%;
   height: 100%;
   z-index: 10;
   opacity: 1;
-  will-change: opacity;
-  -moz-transition: opacity 0.5s;
+  visibility: visible;
+  will-change: opacity, visibility;
+  transition: visibility 0s, opacity 0.5s;
 }
 #videoControls.hidden {
   opacity: 0;
-}
-
-#videoControls.hidden > * {
-  pointer-events: none;
+  visibility: hidden;
+  transition-delay: 0.5s, 0s;
 }
```

Comment:
TBD as to whether these changes are necessary based on whether 'hide-controls' element is necessary.